src/pages/multichain.tsx

```diff
@@ -176,11 +177,9 @@ function MultichainPage() {
                 Trustless multi-chain on ICP
               </h1>
               <p className="tw-lead-sm md:tw-lead mb-0 animate-fade-up animation-delay-200">
-                Decentralized applications and services can delight users by
-                incorporating the assets and functionality of multiple
-                blockchains. Using the Internet Computer, developers gain the
-                technical superpowers necessary to deliver such game-changing
-                experiences, while preserving user safety and convenience.
+              Unique in blockchain, canister smart contracts on the Internet Computer can talk to other chains.
+              This is because of the innovative Chain Key cryptography that allows for secure, decentralized, communication.          
```

Comment:
```suggestion
              This is because of the innovative chain-key cryptography that allows for secure, decentralized, communication.          
```